src/model_theory/basic.lean

```diff
@@ -130,6 +133,11 @@ instance : has_coe_t L.const M :=
 lemma fun_map_eq_coe_const {c : L.const} {x : fin 0 → M} :
   fun_map c x = c := congr rfl (funext fin.elim0)
 
+/-- This throws errors as an instance, but could be useful for showing that a particular class of
+structures must be nonempty. -/
```

Comment:
Please describe the statement first, then explain why this can't be an instance.

Reply:
I've done my best to diagnose it. I believe the problem is that the assumptions depend on `L` but the conclusions do not. It however may still be useful later for a particular language.